src/content/1.7/scale/_index.md

```diff
@@ -7,6 +7,34 @@ chapter: true
 
 ### Chapter 8
 
-# How to make PrestaShop scale
+# Scale
 
-Follow this guide to optimize your PrestaShop shop.
+Scaling your PrestaShop instalation is something we hope every merchant will need, because it means her business is growing.
+
+Hopefully, there are plenty advices in these pages that will help.
+
+# Scalings
+
+Before doing anything to your PrestaShop setup, there are several things you need to know.
+
+First, to be aware there are several ways to scale any application, PrestaShop incuded.
+
+## Vertical scaling
+
+The most used one is called vertical scaling, where you boost your server's performance : you add some RAM, some CPU, increase the disks performances, and so on. It can be quick and easy if your hoster allows it and does not require any change on your application setup and configuration.
```

Comment:
```suggestion
The most used one is called vertical scaling, where you boost your server's performance: you add some RAM, some CPU, increase the IO performance, and so on. It can be quick and easy if your hoster allows it and does not require any change in your application setup and configuration.
```